src/Register.php

```diff
@@ -11,7 +11,7 @@ class Register {
 	 *
 	 * @var array
 	 */
-	private static $_registered_widgets = [];
+	private static $_widgets = [];
```

Comment:
Read here:

https://github.com/moxie-lean/wp-widgets/pull/23#issue-156397580 <https://github.com/moxie-lean/wp-widgets/pull/23#issue-156397580>

> On Jun 1, 2016, at 8:15 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In src/Register.php <https://github.com/moxie-lean/wp-widgets/pull/23#discussion_r65357132>:
> 
> > @@ -11,7 +11,7 @@ class Register {
> >  	 *
> >  	 * @var array
> >  	 */
> > -	private static $_registered_widgets = [];
> > +	private static $_widgets = [];
> Why change this variable name?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/moxie-lean/wp-widgets/pull/23/files/da2b2163ee75d6c22433b4221b9e2f763c6e4480#r65357132>, or mute the thread <https://github.com/notifications/unsubscribe/ADvViTagZHQdJ7rjuccbi4zCzrAN08vYks5qHYXYgaJpZM4IlAs3>.
>